on-chain-metrics.md

```diff
@@ -124,4 +124,114 @@ List of metrics with 5 minutes minimal resolution:
 - age_destroyed
 - exchange_inflow
 - exchange_outflow
-- exchange_balance
\ No newline at end of file
+- exchange_balance
```

Comment:
I think we should also remove this file at some point and point to the query for available metrics + (list only those that are not in available metrics). But lets leave for now.